### ISSUE TITLE
FOP integration test failed in native mode

### ISSUE THREAD

**Issue by ffang** [apache/camel-quarkus#3280]
the exception is like
```
Caused by: java.lang.UnsatisfiedLinkError: no awt in java.library.path
	at com.oracle.svm.core.jdk.NativeLibrarySupport.loadLibraryRelative(NativeLibrarySupport.java:132)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:197)
	at java.lang.Runtime.loadLibrary0(Runtime.java:830)
	at java.lang.System.loadLibrary(System.java:1871)
	at java.awt.Toolkit$3.run(Toolkit.java:1395)
	at java.awt.Toolkit$3.run(Toolkit.java:1393)
	at java.security.AccessController.doPrivileged(AccessController.java:87)
	at java.awt.Toolkit.loadLibraries(Toolkit.java:1392)
	at java.awt.Toolkit.<clinit>(Toolkit.java:1425)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:375)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:295)
	at java.awt.Color.<clinit>(Color.java:275)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.invokeClassInitializer(ClassInitializationInfo.java:375)
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:295)
	at org.apache.xmlgraphics.java2d.GraphicContext.<init>(GraphicContext.java:92)
	at org.apache.fop.render.intermediate.IFGraphicContext.<init>(IFGraphicContext.java:42)
	at org.apache.fop.render.intermediate.IFRenderer.<init>(IFRenderer.java:129)
	at org.apache.fop.render.RendererFactory.createRendererForDocumentHandler(RendererFactory.java:318)
	at org.apache.fop.render.RendererFactory.tryIFDocumentHandlerMaker(RendererFactory.java:299)
	at org.apache.fop.render.RendererFactory.createRenderer(RendererFactory.java:279)
	at org.apache.fop.area.RenderPagesModel.<init>(RenderPagesModel.java:71)
	at org.apache.fop.area.AreaTreeHandler.setupModel(AreaTreeHandler.java:135)
	at org.apache.fop.area.AreaTreeHandler.<init>(AreaTreeHandler.java:105)
	at org.apache.fop.render.RendererFactory.createFOEventHandler(RendererFactory.java:363)
	at org.apache.fop.fo.FOTreeBuilder.<init>(FOTreeBuilder.java:107)
	at org.apache.fop.apps.Fop.createDefaultHandler(Fop.java:104)
	at org.apache.fop.apps.Fop.<init>(Fop.java:78)
	at org.apache.fop.apps.FOUserAgent.newFop(FOUserAgent.java:184)
	at org.apache.fop.apps.FopFactory.newFop(FopFactory.java:246)
	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:92)
	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
	... 39 more
```

```
Caused by: java.lang.NoClassDefFoundError: Could not initialize class java.awt.Color
	at com.oracle.svm.core.classinitialization.ClassInitializationInfo.initialize(ClassInitializationInfo.java:239)
	at org.apache.xmlgraphics.java2d.GraphicContext.<init>(GraphicContext.java:92)
	at org.apache.fop.render.intermediate.IFGraphicContext.<init>(IFGraphicContext.java:42)
	at org.apache.fop.render.intermediate.IFRenderer.<init>(IFRenderer.java:129)
	at org.apache.fop.render.RendererFactory.createRendererForDocumentHandler(RendererFactory.java:318)
	at org.apache.fop.render.RendererFactory.tryIFDocumentHandlerMaker(RendererFactory.java:299)
	at org.apache.fop.render.RendererFactory.createRenderer(RendererFactory.java:279)
	at org.apache.fop.area.RenderPagesModel.<init>(RenderPagesModel.java:71)
	at org.apache.fop.area.AreaTreeHandler.setupModel(AreaTreeHandler.java:135)
	at org.apache.fop.area.AreaTreeHandler.<init>(AreaTreeHandler.java:105)
	at org.apache.fop.render.RendererFactory.createFOEventHandler(RendererFactory.java:363)
	at org.apache.fop.fo.FOTreeBuilder.<init>(FOTreeBuilder.java:107)
	at org.apache.fop.apps.Fop.createDefaultHandler(Fop.java:104)
	at org.apache.fop.apps.Fop.<init>(Fop.java:78)
	at org.apache.fop.apps.FOUserAgent.newFop(FOUserAgent.java:184)
	at org.apache.fop.apps.FopFactory.newFop(FopFactory.java:246)
	at org.apache.camel.component.fop.FopProducer.transform(FopProducer.java:92)
	at org.apache.camel.component.fop.FopProducer.process(FopProducer.java:66)
	at org.apache.camel.support.AsyncProcessorConverterHelper$ProcessorToAsyncProcessorBridge.process(AsyncProcessorConverterHelper.java:66)
	... 39 more
```

**Comment by ffang** [comment]
This is probably mac machine specific, since per the graalvm issue comment here
https://github.com/oracle/graal/issues/2842#issuecomment-879620758
AWT isn't supported on macos yet.

I can see this problem on both main and camel-main branch.

Weird thing is that I passed all native tests on my mac weeks ago, not sure if this is caused by mac env mess-up(I installed graalvm version with JDK17 also, but this error is reproduced with JDK11 on my mac).

Anyway, I think we better disable this test on mac if this is mac specific
